### PR TITLE
add vortex gui to welcome page

### PR DESCRIPTION
**Pull request by suhailsaqan**
@benthecarman What other text should I add?

Screenshot 2022-10-01 at 12 03 04 PM

### CODE REVIEW

**Comment by deregs** [comment]
Just added a more updated screenshot in the readme that you could use here

File: website/src/pages/index.js [hunk 1]
```diff
@@ -135,6 +135,28 @@ function Home() {
...
+                    <div className="container">
+                        <div className="row">
+                            <div style={{marginTop: '100px'}} className={classnames('col col--6 text--center', styles.feature)}>
+                                <h2>Vortex GUI</h2>
+                                <p>
+                                    Vortex GUI is an application used for interacting with the Vortex Client.
+                                </p>
+                                <p>
+                                    Download <span className="learn-more"><a href="https://github.com/ln-vortex/ln-vortex-gui"> here →</a></span>
+                                </p>
```
Comment (benthecarman) [inline website/src/pages/index.js:148]: I think instead should have something like:

```html
<h2>Easy to use</h2>

<p>
 Vortex comes with an easy to use GUI that is usable as a desktop application or as web server.
 Vortex handles all the hard work and privacy is only a click away.
</p>

```

**Review by benthecarman — CHANGES_REQUESTED** [review]
should update the image to be the one from the other PR

**Review by benthecarman — CHANGES_REQUESTED** [review]
Still need to update the image

**Pull request merged**